src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSpark.java

```diff
@@ -56,43 +58,99 @@
               optional  = false)
     public String outDirPrefix = null;
 
-    @Argument(doc       = "To run k-mer based read correction, filter and duplication removal in SGA or not, with default parameters.",
+    @Argument(doc       = "To run k-mer based read correction with default parameters (as provided by SGA) or not.",
               shortName = "correct",
-              fullName  = "correctNFilter",
+              fullName  = "kmerCorrectReads",
               optional  = true)
-    public boolean runCorrectionSteps = false;
+    public boolean runCorrection = false;
 
     @Argument(doc       = "If true, capture the stdout and stderr of the SGA processes along with the assembly output (valuable for debugging but hurts performance)",
               shortName = "captureStdIO",
               fullName  = "captureSGAStdOutAndError",
               optional  = true)
     public boolean enableSTDIOCapture = false;
 
+    @Argument(doc       = "An estimate on the number of fastq files in the input directory to be fed to the tool",
```

Comment:
Instead of requiring an estimate of the number of files, could the tool count them? Seems like it should be possible. Maybe like this:

````
final FileSystem hadoopFileSystem = FileSystem.get(ctx.hadoopConfiguration());
hadoopFileSystem.listFiles(new org.apache.hadoop.fs.Path(pathToAllInterleavedFASTQFiles), false);
````

Then this parameter could go away.